tests/abi/tools/revng-check-compatibility-with-abi/Verify.cpp

```diff
@@ -92,7 +89,7 @@ void VH::verify(const abi::FunctionType::Layout &FunctionLayout,
         std::size_t CurrentSize = ExtractedStackBytes.size();
         ExtractedStackBytes.resize(CurrentSize + Argument.Stack->Size);
```

Comment:
Remove all usages of resize. Early optimization.

Reply:
I'd argue that it's helpful semantically to say: "hey, this next vector is pretty much an array with X elements), but I don't feel like we need to have that conversation today. Will remove.

Reply:
Ah, wait, I was wrong. This is not an optimization: it's used for shrinking here!

Do you want an assert kind of like
```cpp
revng_assert(CurrentSize + Argument.Stack->Size > ExtractedStackBytes.size());
```
to bring that point home?

Reply:
What's wrong with `back_inserter`?
Please, do not try to make the code performant unless it really matters.

Reply:
If you want, you can make a helper function to put in some public header that can be reused. Otherwise, let's go with back inserter.

Reply:
Done.